### PR TITLE
Changing research board to editorial board

### PR DESCRIPTION
**Pull request by taddyb**
Addressing issue #11, and adding a quick walkthrough for someone making their first blog post. Also changed one of the research goals in the readme to be more specific.

image

See `index.md` for changes.

### CODE REVIEW

File: docs/roadmap/index.md
Comment (jmframe) [inline docs/roadmap/index.md:1]: This isn't in the scope of this PR. But I wonder if we want to specify "weekly"? Does that mean we can't get post more submissions than that? Does it mean we have to have a submission each week?

**Review by jmframe — APPROVED** [review]
thanks!

File: docs/roadmap/index.md
Comment (taddyb) [reply docs/roadmap/index.md:1]: I think this is a good point. Maybe consistently updated is a better descriptor

**Comment by rappjer1** [comment]
Looks good, I imagine instructions will be one of those evolving documents as we encounter more questions/issues

**Review by rappjer1 — APPROVED** [review]

**Pull request merged**